Rakefile

```diff
@@ -19,4 +19,36 @@ desc 'Generate code coverage'
 RSpec::Core::RakeTask.new(:coverage) do |t|
   t.rcov = true
   t.rcov_opts = ['--exclude', 'spec']
-end
\ No newline at end of file
+end
+
+desc 'Executes reference tests (agent only) intended for use in CI'
+task :reference_tests do
+    command =<<-EOS
+bundle exec beaker                          \
+    --debug                                 \
+    --preserve-hosts never                  \
+    --config tests/configs/$PLATFORM        \
```

Comment:
Let's make sure the task will fail with a descriptive message if the $PLATFORM envar is not set.